packages/common/http/src/client.ts

```diff
@@ -101,9 +101,9 @@ export class HttpClient {
   constructor(private handler: HttpHandler) {}
 
   /**
-   * Sends an `HTTPRequest` and returns a stream of `HTTPEvents`.
+   * Sends an `HttpRequest` and returns a stream of `HttpEvents`.
```

Comment:
Actually, there is no `HttpEvents` type (only [HttpEvent][1]). Let's change this to:

```suggestion
   * Sends an `HttpRequest` and returns a stream of `HttpEvent`s.
```

(Same for other occurrences of `HttpEvents` below.)

[1]: https://angular.io/api/common/http/HttpEvent